### DISCUSSION TITLE
Issue with one broker getting killed

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-operator#8663] [General]
GKE 1.25+
Strimzi 0.34

We have seen that in three different env  , sporadically one BROKER ( KAFKA-0) in all cases. Dies after being healthy for days.  Operator log says failing to reconcile to kafka-0 as timeout occured (the usual message).  But never try to restart the kafka-0.  is this something seen before. its not in my cluster in our organization, so I am waiting for getting access to see what is happening. thought I will run pass the expert if anyone has seen anything similar.

**Comment by sreejesh-radhakrishnan-db** [comment]
Have looked in GKE cluster log, Strmzi log and generally all events when first time Strimzi reconciliation failed.  and We cannot see any error on OPERATOR log, other than failing reconciliation java error of timeout.  We are seeing this issue only when we run STRIMZI POD and Not happening when we use KAFKA as Statefulset. Any similar scenario have you seen?

**Reply by scholzj** [reply]
You will have to share some logs and configurations. It is also not clear what `STRIMZI POD` is - the cluster operator?
